pex/bin/pex.py

```diff
@@ -519,6 +519,20 @@ def configure_clp_pex_environment(parser):
         "if only sdists or non-matching binary distributions can be found.",
     )
 
+    group.add_option(
+        "--use-first-matching-interpreter",
+        dest="use_first_matching_interpreter",
+        default=False,
+        action="callback",
+        callback=parse_bool,
+        help=(
+            "If multiple interpreters are valid, use the first one. Normally, when multiple "
+            "interpreters match, Pex will resolve requirements for each interpreter; this allows "
+            "the resulting Pex to work with more interpreters, such as different Python versions. "
+            "However, resolving for multiple interpreters results in worse performance."
```

Comment:
I struggled writing this. I wasn't sure how much detail to give about how Pex does interpreter matching, which is via a combination of `--python`, `--interpreter-constraint`, and `--platforms` (but only if `--resolve-local-platforms` is enabled).

I also wasn't sure if we should suggest when you might want this and when you might not care.

Reply:
I guess this should clarify what "first one" means. "First one it happens to encounter" or "Lowest version"?